app/src/main/java/xyz/jekyllex/ui/activities/settings/SettingsActivity.kt

```diff
@@ -194,6 +191,23 @@ fun SettingsView() {
                     },
                 )
 
+                sliderPreference(
+                    valueSteps = 10,
+                    defaultValue = DEBOUNCE_DELAY.defaultValue.get(),
+                    valueRange = .25f..3f,
+                    key = DEBOUNCE_DELAY.key,
+                    valueText = { Text(text = "%.2fs".format(it)) },
+                    title = { Text(context.getString(R.string.debounce_setting_title)) },
+                    summary = { Text(context.getString(R.string.debounce_setting_summary)) },
+                )
+
+                listPreference(
+                    key = EDITOR_THEME.key,
+                    values = themeMap.keys.toList(),
+                    title = { Text(context.getString(R.string.theme_setting_title)) },
+                    defaultValue = EDITOR_THEME.defaultValue.get()
+                )
+
```

Comment:
Let's move this right below the `editor_settings` `preferenceCategory`. Also, plase add `valueToText` & `summary` to make it easier to know theme names.

```kotlin
 listPreference(
    key = EDITOR_THEME.key,
    values = themeMap.keys.toList(),
    defaultValue = EDITOR_THEME.defaultValue.get(),
    summary = { Text(themeMap[it] ?: "Unknown theme") },
    valueToText = { buildAnnotatedString { append(themeMap[it]) }},
    title = { Text(context.getString(R.string.theme_setting_title)) },
)
```